.github/labeler.yml

```diff
@@ -0,0 +1,172 @@
+ # trademarks, maskwork rights, or any other form of intellectual
+ # property whatsoever. Maxim Integrated Products, Inc. retains all
+ # ownership rights.
+ #
+ ###############################################################################
+
+'BLE':
+- 'Examples/*/BLE*/*'
+- 'Libraries/BlePhy/*'
+- 'Libraries/Cordio/*'
+- 'Tools/Bluetooth/*'
+
+'documentation':
+- 'Documentation/*'
+
+'MAX32520':
+- 'Examples/MAX32520/*'
+- '*es17.*'
+- 'Libraries/Boards/MAX32520/*'
+- 'Libraries/CMSIS/Device/Maxim/MAX32520/*'
+- 'Libraries/PeriphDrivers/max32520_files.mk'
+- 'Libraries/Include/MAX32520/*'
+
+'MAX32570':
+- '*me13.*'
+- 'Libraries/Boards/MAX32570/*'
+- 'Libraries/CMSIS/Device/Maxim/MAX32570/*'
+- 'Libraries/PeriphDrivers/max32570_files.mk'
+- 'Libraries/Include/MAX32570/*'
+
+'MAX32650':
+- 'Examples/MAX32650/*'
+- '*me10.*'
+- 'Libraries/Boards/MAX32650/*'
+- 'Libraries/CMSIS/Device/Maxim/MAX32650/*'
+- 'Libraries/PeriphDrivers/max32650_files.mk'
+- 'Libraries/Include/MAX32650/*'
+
+'MAX32655':
+- 'Examples/MAX32655/*'
+- '*me17.*'
+- 'Libraries/Boards/MAX32655/*'
+- 'Libraries/CMSIS/Device/Maxim/MAX32655/*'
+- 'Libraries/PeriphDrivers/max32655_files.mk'
+- 'Libraries/Include/MAX32655/*'
+
+'MAX32660':
+- 'Examples/MAX32660/*'
+- '*me11.*'
+- 'Libraries/Boards/MAX32660/*'
+- 'Libraries/CMSIS/Device/Maxim/MAX32660/*'
+- 'Libraries/PeriphDrivers/max32660_files.mk'
+- 'Libraries/Include/MAX32660/*'
+
+'MAX32662':
+- 'Examples/MAX32662/*'
+- '*me12.*'
+- 'Libraries/Boards/MAX32662/*'
+- 'Libraries/CMSIS/Device/Maxim/MAX32662/*'
+- 'Libraries/PeriphDrivers/max32662_files.mk'
+- 'Libraries/Include/MAX32662/*'
+
+'MAX32665':
+- 'Examples/MAX32665/*'
+- '*me14.*'
+- 'Libraries/Boards/MAX32665/*'
+- 'Libraries/CMSIS/Device/Maxim/MAX32665/*'
+- 'Libraries/PeriphDrivers/max32665_files.mk'
+- 'Libraries/Include/MAX32665/*'
+
+'MAX32670':
+- 'Examples/MAX32670/*'
+- '*me15.*'
+- 'Libraries/Boards/MAX32670/*'
+- 'Libraries/CMSIS/Device/Maxim/MAX32670/*'
+- 'Libraries/PeriphDrivers/max32670_files.mk'
+- 'Libraries/Include/MAX32670/*'
+
+'MAX32672':
+- 'Examples/MAX32672/*'
+- '*me21.*'
+- 'Libraries/Boards/MAX32672/*'
+- 'Libraries/CMSIS/Device/Maxim/MAX32672/*'
+- 'Libraries/PeriphDrivers/max32672_files.mk'
+- 'Libraries/Include/MAX32672/*'
+
+'MAX32675':
+- 'Examples/MAX32675/*'
+- '*me16.*'
+- 'Libraries/Boards/MAX32675/*'
+- 'Libraries/CMSIS/Device/Maxim/MAX32675/*'
+- 'Libraries/PeriphDrivers/max32675_files.mk'
+- 'Libraries/Include/MAX32675/*'
+
+'MAX32680':
+- 'Examples/MAX32680/*'
+- '*me20.*'
+- 'Libraries/Boards/MAX32680/*'
+- 'Libraries/CMSIS/Device/Maxim/MAX32680/*'
+- 'Libraries/PeriphDrivers/max32680_files.mk'
+- 'Libraries/Include/MAX32680/*'
+
+'MAX32690':
+- 'Examples/MAX32690/*'
+- '*me18.*'
+- 'Libraries/Boards/MAX32690/*'
+- 'Libraries/CMSIS/Device/Maxim/MAX32690/*'
+- 'Libraries/PeriphDrivers/max32690_files.mk'
+- 'Libraries/Include/MAX32690/*'
+
+'MAX78000':
+- 'Examples/MAX78000/*'
```

Comment:
The MAX78000 uses some of the *me17* files.  (At least it did once upon a time.)

Reply:
You're right, so does the ME20 -> ME17 and ME16 -> ME15. I've specified the filtering based on the SVD and {part}_files.mk file.